src/telegram.ts

```diff
@@ -41,6 +41,9 @@ class Telegram extends ApiClient {
     return `${this.options.apiRoot}/file/bot${this.token}/${fileId.file_path}`
   }
 
+  /**
+   * @deprecated use `Telegraf::launch`
+   */
```

Comment:
Why adding `@depecated`? If a user what's to get the raw updates directly for whatever, this should be supported.

However, I do agree that we should add a comment here, explaining why this method is rarely useful. It must not become a pitfall for new users, so the method should never be called by accident.

Reply:
My logic was, it's not used internally anymore, and probably shouldn't be used by users either. Users can use well-typed `Telegram::callApi` if they want to proceed anyway.

Reply:
I undeprecated it already, should I un-undeprecate? :)

Reply:
No, I still think that it should not be deprecated, but I understood your point now.

I suggest to leave it undeprecated